ci/docker_cache.py

```diff
@@ -37,7 +37,7 @@
 DOCKERHUB_LOGIN_NUM_RETRIES = 5
 DOCKERHUB_RETRY_SECONDS = 5
 DOCKER_CACHE_NUM_RETRIES = 3
-DOCKER_CACHE_TIMEOUT_MINS = 5
+DOCKER_CACHE_TIMEOUT_MINS = 15
```

Comment:
Why increase this timeout? (It's the timeout for pulling the docker cache from dockerhub, which normally takes 30s, I haven't seen it take more than 3min)

Reply:
Please check:

http://jenkins.mxnet-ci.amazon-ml.com/job/restricted-docker-cache-refresh/job/master/1190/console

Otherwise why would I change it?

Reply:
Did you monitor the network and ebs metrics during that execution? Jose and I have made the experience that we have a huge download spike at the beginning (which is good), followed up by a super long hang which then results in the timeout. The problem there is that hang and increasing the timeout won't solve it but just delay the failure.

Reply:
What you described is a result of the big amount of parallel requests made to the docker daemon and is exactly what this PR addresses. I think you don't understand the root cause of the problem based on your comment and suggestions. Let me know if you want to have a call so I can explain the problem in detail. There's also graphs that corroborate what I'm describing, I could attach them to this PR.